text/0012-frameworks.md

```diff
@@ -0,0 +1,52 @@
+# Frameworks at Canopy
+- RFC PR: https://github.com/CanopyTax/frontend-rfc/pull/12
+
+## Definition
+There are a lot of front-end frameworks out there and we use several at Canopy. By design we've sought to avoid locking ourselves
```

Comment:
The word framework can mean a lot of things. I think it would be good to define that we're talking about "UI frameworks" or "application frameworks" or something like that? To clarify that we are talking about React/Angular/Vue/etc, not jest/css modules/etc which are sometimes called frameworks.

> A UI framework is a library that creates dom elements (e.g. React, Angular, or Vue). Nowadays, most frameworks have an implementation of a "component" building block for creating user interfaces.

> Libraries that do not create dom elements, such as bundlers, testing libraries, css libraries, or compilers, are not covered by this RFC

Reply:
Also, the second and third sentences in this section feel more like a Reason for Decision section than a Definition section